Jenkinsfile

```diff
@@ -1 +1,8 @@
-buildPlugin()
+def builds = [:]
+builds['docker'] = { buildPlugin(platforms: ['docker']) }
+builds['windows'] = {
+    withEnv(['SKIP_BINARY_GENERATION=true']) {
+        buildPlugin(platforms: ['windows'])
+    }
+}
+parallel builds
```

Comment:
Yeah, it does look fine when you start it up, you only see two branches. However, when it finishes, you always see 4 branches.

The only reason I had it this way was because I thought I would have to also check for the OS type of the platform as well, since both platforms would get the environment variable.  I was not sure if there was a clean way to do that in the pom